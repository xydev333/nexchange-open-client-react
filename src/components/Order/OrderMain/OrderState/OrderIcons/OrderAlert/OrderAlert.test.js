import React from 'react';
import { shallow } from 'enzyme';
import OrderAlert from './OrderAlert';

describe('OrderAlert', () => {
  it('renders correctly', () => {
    expect(shallow(<OrderAlert />)).toMatchSnapshot();
  });
});
