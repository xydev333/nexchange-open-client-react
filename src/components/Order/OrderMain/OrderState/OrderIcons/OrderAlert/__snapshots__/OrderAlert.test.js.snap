// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderAlert renders correctly 1`] = `ShallowWrapper {}`;
